SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -104,6 +103,7 @@ message DataMessage {
   optional GroupContext       group       = 3;
   optional uint32             flags       = 4;
   optional uint32             expireTimer = 5;
+  optional bytes              profileKey  = 6;
```

Comment:
Note that I'm sending the profile key with _NO_ null messages, _SOME_ data and call messages and _ALL_ sync messages (since we always want to sync our profile key with linked devices.